src/home-assistant.html

```diff
@@ -129,21 +129,27 @@
         return conn.callService(domain, service, serviceData || {})
           .then(function () {
             var message;
+            var name;
+            if (serviceData.entity_id && this.hass.states &&
+                this.hass.states[serviceData.entity_id] &&
+                this.hass.states[serviceData.entity_id].attributes) {
+              name = this.hass.states[serviceData.entity_id].attributes.friendly_name;
```

Comment:
Not all states will have a friendly name attribute. Please use `window.hassUtil.computeStateName`